sdk/identity/azure-identity/README.md

```diff
@@ -1,15 +1,11 @@
 # Azure Identity client library for Python
 
-The Azure Identity library provides [Azure Active Directory (AAD)](https://docs.microsoft.com/azure/active-directory/fundamentals/active-directory-whatis) token authentication through a set of convenient TokenCredential implementations. It enables Azure SDK clients to authenticate with AAD, while also allowing other Python apps to authenticate with AAD work and school accounts, Microsoft personal accounts (MSA), and other Identity providers like [AAD B2C](https://docs.microsoft.com/azure/active-directory-b2c/overview) service.
+The Azure Identity library provides [Azure Active Directory (Azure AD)](https://learn.microsoft.com/azure/active-directory/fundamentals/active-directory-whatis) token authentication through a set of convenient TokenCredential implementations. It enables Azure SDK clients to authenticate with Azure AD, while also allowing other Python apps to authenticate with Azure AD work and school accounts, Microsoft personal accounts (MSA), and other Identity providers like [Azure AD B2C](https://learn.microsoft.com/azure/active-directory-b2c/overview) service.
```

Comment:
@scottaddie I noticed that at the end, we say "...and other Identity providers like [Azure AD B2C]..." when in the known issues section, we say "This library doesn't support [Azure AD B2C]...". Is there anything else that can be called out here ("...and other Identity providers like [something else]") or would just saying "...and other Identity providers." suffice?